### PR TITLE
add jenkins project for kubenet

### PR DESCRIPTION
**Pull request by freehan**
added a jenkins project for gce using kubenet as network provider

`k8s-jkns-e2e-gce-kubenet` has been created and configured

### CODE REVIEW

**Comment by ixdy** [comment]
what's up with the CLA?

also cc @spxtr

**Comment by freehan** [comment]
I have no idea. I emailed folks. No response yet. 

It should not be a blocker though. We can by pass manually.

**Comment by freehan** [comment]
PTAL

**Comment by freehan** [comment]
@googlebot I have signed it!

**Comment by freehan** [comment]
@ixdy PTAL

**Comment by spxtr** [comment]
LGTM

File: hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml [hunk 1]
```diff
@@ -155,6 +155,16 @@
                 export MULTIZONE="true"
                 export E2E_ZONES="us-central1-a us-central1-b us-central1-f" # Where the nodes reside.  Master is in the first one.
                 export KUBE_GCE_ZONE="us-central1-a" # Where the master resides - hangover due to legacy scripts.
+        - 'gce-kubenet':
+            description: 'Run all non-flaky, non-slow, non-disruptive, non-feature tests on GCE using kubenet as network provider'
+            timeout: 120
```
Comment (ixdy) [inline hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml]: this timeout seems a bit excessive, but I guess we can dial it back later if necessary.

**Comment by freehan** [comment]
etcd problem: 

```console
scheduler_test.go:252: Test 0: Pod did not get scheduled on an unschedulable node scheduler_test.go:265: Test 0: failed to schedule a pod: timed out waiting for the condition scheduler_test.go:272: Failed to delete pod: client: etcd cluster is unavailable or misconfigured scheduler_test.go:276: Failed to delete node: client: etcd cluster is unavailable or misconfigured scheduler_test.go:225: Failed to create node: client: etcd cluster is unavailable or misconfigured
```

**Comment by freehan** [comment]
@k8s-bot unit test this

**Pull request merged**